agent/client/client.go

```diff
@@ -459,6 +353,116 @@ func (c *Client) processChannelRequests(ctx context.Context) {
 	c.l.Debug("Channel closed.")
 }
 
+func (c *Client) handleStartActionRequest(p *agentpb.StartActionRequest) error {
+	timeout := p.Timeout.AsDuration()
+	if err := p.Timeout.CheckValid(); err != nil {
+		timeout = 0
```

Comment:
Same as it was before: https://github.com/percona/pmm/blob/3dd4291df828af85ec886dd0a1cfb05c6f2a5222/agent/client/client.go#L565-L576

Original implementation for some reason throw error for zero timeout, IMHO we can use default value for that case as well

Reply:
Timeout value has this comment: https://github.com/percona/pmm/blob/3dd4291df828af85ec886dd0a1cfb05c6f2a5222/agent/client/client.go#L52

I guess better to keep fallback to default in case of error, it seems that it's important for backward compatibility

Reply:
> the original implementation didn't throw an error.
Ah, yep, my bad. But anyway, behavior left the same. In case of error or zero timeout we are using default. Right?

Reply:
Yes, we use default

Reply:
Ok, that's how it works in this PR